recipe/meta.yaml

```diff
@@ -1,17 +1,16 @@
-{% set version = "1.5.1" %}
-{% set eccodes_version = "2.26.0" %}
+{% set version = "1.5.2" %}
+{% set eccodes_version = "2.30.0" %}
```

Comment:
Upstream says the "latest" version of `eccodes` is required and this was the latest version as of the `eccodes-python` 1.5.2 release.